OpenQA/Qemu/Proc.pm

```diff
@@ -56,6 +60,11 @@ has snapshot_conf   => sub { return OpenQA::Qemu::SnapshotConf->new(); };
 sub new {
     my $self = shift->SUPER::new(@_);
 
+    $self->_process(process(
+            pidfile       => 'qemu.pid',
+            separate_err  => 0,
+            blocking_stop => 1)
+    );
```

Comment:
can you create this lazily instead of in new?

Reply:
Sure

Reply:
But next step, would be proper inheritance here (also in Qemu::Proc)